config/busola/telemetry_busola_extension_cm.yaml

```diff
@@ -14,14 +14,22 @@ data:
         widget: Badge
 
     body:
-      - name: Trace
+      - name: Tracing
```

Comment:
Tracing vs Metrics? Shouldn't it be Traces vs Metrics? From naming perspective I thought we are shifting away from Logging/Tracing/Monitoring to Logs/Traces/Metrics

Reply:
Good point, I created a follow-up PR: https://github.com/kyma-project/telemetry-manager/pull/1050